app/assets/stylesheets/logixRelease.css

```diff
@@ -129,4 +129,9 @@ a.btn.btn-outline-primary {
 
 a.btn.btn-outline-danger {
     margin-top: 10px;
-}
\ No newline at end of file
+}
+
+.card-shadow:hover {
+    box-shadow: 4px 4px #dfdfdf;
+    transition: box-shadow 0.4s ease-in;
```

Comment:
I guess transform property is used with `psedu selectors` like `:hover` , transition property is used with `.class-name`

Reply:
> I guess transform property is used with `psedu selectors` like `:hover` , transition property is used with `.class-name`

True 👍 . Please make the requested changes @SamriddhiMishra .

Reply:
Sorry! It was working fine with :hover so I made the changes.
![After3](https://user-images.githubusercontent.com/43378325/75041992-b3425680-54e3-11ea-928a-09eccb02c166.gif)

But I will change that in a minute.

Reply:
My bad. Thanks!